extensions/windows-event-log/tests/MetadataWalkerTests.cpp

```diff
@@ -74,7 +74,7 @@ const short event_type_index = 178;  // NOLINT short comes from WINDOWS API
 
 class FakeWindowsEventLogMetadata : public WindowsEventLogMetadata {
  public:
-  [[nodiscard]] std::string getEventData(EVT_FORMAT_MESSAGE_FLAGS flags) const override { return "event_data_for_flag_" + std::to_string(flags); }
+  [[nodiscard]] nonstd::expected<std::string, std::error_code> getEventData(EVT_FORMAT_MESSAGE_FLAGS flags) const noexcept override { return "event_data_for_flag_" + std::to_string(flags); }
```

Comment:
Changed this to `noexcept`. There are allocations here. If they fail, the test will terminate and report failure, which is an acceptable behavior IMO.